easybuild/easyblocks/generic/rpm.py

```diff
@@ -45,6 +45,7 @@
 from easybuild.easyblocks.generic.binary import Binary
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import which
 from easybuild.tools.run import run_cmd
 from easybuild.tools.systemtools import check_os_dependency
```

Comment:
It is needed here in line 125. But I think that one can be also replaced by `which`. Would you agree?